http4k-contract-ui/swagger/src/main/kotlin/org/http4k/contract/ui/swagger/swaggerUiWebjar.kt

```diff
@@ -2,18 +2,28 @@ package org.http4k.contract.ui.swagger
 
 import org.http4k.contract.ui.SwaggerUiConfig
 import org.http4k.contract.ui.toFilter
+import org.http4k.core.Method
+import org.http4k.core.Response
+import org.http4k.core.Status
+import org.http4k.core.appendToPath
 import org.http4k.core.then
+import org.http4k.core.with
+import org.http4k.lens.Header
 import org.http4k.routing.ResourceLoader
+import org.http4k.routing.bind
 import org.http4k.routing.routes
 import org.http4k.routing.static
 
 /**
  * Serve locally hosted Swagger UI
  */
-fun swaggerUiWebjar(configFn: SwaggerUiConfig.() -> Unit = {}) = SwaggerUiConfig()
-    .also(configFn)
-    .toFilter()
-    .then(routes(
-        static(ResourceLoader.Classpath("org/http4k/contract/ui/public-config/")),
-        static(ResourceLoader.Classpath( "/META-INF/resources/webjars/swagger-ui/4.18.1"))
-    ))
+fun swaggerUiWebjar(configFn: SwaggerUiConfig.() -> Unit = {}) = routes(
+    "" bind Method.GET to { Response(Status.FOUND).with(Header.LOCATION of it.uri.appendToPath("index.html")) },
+
+    SwaggerUiConfig()
```

Comment:
The `SwaggerUiConfig` filter only needs to apply to the config Javascript.  This should be a performance improvement.